Keas.Mvc/Views/TeamAdmin/Upload.cshtml

```diff
@@ -1,78 +1,194 @@
+                                if (!item.Success)
+                                { style = "background-color: lightcoral"; }
+                                <tr style="@style">
+                                    <td>@item.LineNumber</td>
+                                    <td>
+                                        @item.Success.ToYesNoString()
+                                    </td>
+                                    <td style="word-wrap: break-word; word-break: break-all; white-space: normal;">
+                                        @Html.DisplayFor(a => item.Import.KeyName)
+                                    </td>
+                                    <td>
+                                        @Html.DisplayFor(a => item.Import.KeyCode)
+                                    </td>
+                                    <td>
+                                        @Html.DisplayFor(a => item.Import.SerialNumber)
+                                    </td>
+                                    <td>
+                                        @Html.DisplayFor(a => item.Import.KerbUser)
+                                    </td>
+                                    <td>
+                                        @item.Import.DateIssued.Format("d")
+                                    </td>
+                                    <td>
+                                        @item.Import.DateDue.Format("d")
+                                    </td>
+                                    <td>
+                                        @Html.DisplayFor(a => item.Import.Status)
+                                    </td>
+
+                                    <th>
+                                        @string.Join(", ", item.Messages)
+                                    </th>
+                                    <th>
+                                        @string.Join(", ", item.ErrorMessage)
+                                    </th>
+                                </tr>
+                            }
+
+                        </tbody>
+                    </table>
+                </div>
+            </div>
+        </div>
+    }
+</div>
+
+@section Scripts {
+    @{await Html.RenderPartialAsync("_ValidationScriptsPartial");}
+    <script type="text/javascript" src="https://cdn.datatables.net/1.10.13/js/jquery.dataTables.min.js"></script>
+    <script type="text/javascript" src="https://cdn.datatables.net/buttons/1.5.2/js/buttons.html5.min.js"></script>
+    <script type="text/javascript" src="https://cdn.datatables.net/buttons/1.5.2/js/dataTables.buttons.min.js"></script>
+    <script type="text/javascript" src="https://cdn.datatables.net/buttons/1.5.2/js/buttons.flash.min.js"></script>
+    <script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jszip/3.1.3/jszip.min.js"></script>
+    <script type="text/javascript" src="https://cdn.datatables.net/buttons/1.5.2/js/buttons.print.min.js"></script>

```

Comment:
dump these into a partial so we have all the extension files in one place.  I think if you look in views/shared james did this with reports partials.